files/image_config/hostcfgd/hostcfgd

```diff
@@ -304,13 +304,49 @@ class HostConfigDaemon:
             add = False
 
         self.iptables.iptables_handler(key, data, add)
+        
+    def feature_status_handler(self, key, data):
+        status_data = self.config_db.get_table('FEATURE')
+        for key in status_data.keys():
+            if not key:
+                syslog.syslog(syslog.LOG_WARNING, "FEATURE key is missing")
+                return
+        status = status_data[key]['status']
+        if not status:
+            syslog.syslog(syslog.LOG_WARNING, "status is missing for {}".format(key))
+            return
+        if status == "enabled":
+            start_cmds=[]
+            start_cmds.append("sudo systemctl enable {}".format(key))
+            start_cmds.append("sudo systemctl start {}".format(key))
+            for cmd in start_cmds:
+                syslog.syslog(syslog.LOG_INFO, "Running cmd - {}".format(cmd))
+                try:
+                    subprocess.check_call(cmd, shell=True)
+                except subprocess.CalledProcessError as err:
+                    syslog.syslog(syslog.LOG_ERR, "{} - failed: return code - {}, output:\n{}"
+                              .format(err.cmd, err.returncode, err.output))
+            syslog.syslog(syslog.LOG_INFO, "Feature '{}' is enabled and started".format(key))
+        if status == "disabled":
```

Comment:
Suggest to make this an "elif", then add an "else" case where you log an error message because you received an unexpected value (not "enabled" or "disabled").